src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -95,163 +94,183 @@ namespace CommandLineInterface {
+
+        std::string prefixOutName = "eplus";
+        app.add_option("-p,--output-prefix", prefixOutName, "Prefix for output file names (default: eplus)")->required(false)->option_text("PRE");
+
+        app.add_flag("-r,--readvars", state.dataGlobal->runReadVars, "Run ReadVarsESO after simulation");
+
+        app.add_flag("-c,--convert", state.dataGlobal->outputEpJSONConversion, "Output IDF->epJSON or epJSON->IDF, dependent on input file type");
+
+        app.add_flag("--convert-only",
+                     state.dataGlobal->outputEpJSONConversionOnly,
+                     "Only convert IDF->epJSON or epJSON->IDF, dependent on input file type. No simulation");
+
+        // TODO: make it an enum instead?
+        std::string suffixType = "L";
+        std::string const suffixHelp = R"help(Suffix style for output file names (default: L)
+   L: Legacy (e.g., eplustbl.csv)
+   C: Capital (e.g., eplusTable.csv)
+   D: Dash (e.g., eplus-table.csv))help";
+        app.add_option("-s,--output-suffix", suffixType, suffixHelp)
+            ->option_text("SUFFIX")
+            ->required(false)
+            ->check(CLI::IsMember({"L", "C", "D"}, CLI::ignore_case));
+
+        // TODO: maybe delay validation to output a better error message?
+        const int MAX_N = static_cast<int>(std::thread::hardware_concurrency());
+        app.add_option("-j,--jobs",
+                       state.dataGlobal->numThread,
+                       "Multi-thread with N threads; 1 thread with no arg. (Currently only for G-Function generation)")
+            ->option_text("N")
+            // ->check(CLI::Range(1, MAX_N)  // Tempted to just do that... much simpler
+            // ->check(CLI::Number)
+            ->transform([MAX_N, &state](std::string input) -> std::string {
+                int number_of_threads = -1;
+                bool const converted = CLI::detail::lexical_cast(input, number_of_threads);
+                if (!converted) {
+                    // CLI::ValidationError
+                    return fmt::format("Argument should be an integer, not '{}'", input);
+                }
+                if (number_of_threads <= 0) {
+                    DisplayString(state, "Invalid value for -j arg. Defaulting to 1.");
+                    return "1";
+                }
+                if (number_of_threads > MAX_N) {
+                    DisplayString(state,
+                                  fmt::format("Invalid value for -j arg. Value exceeds num available. Defaulting to num available. -j {}", MAX_N));
+                    return std::to_string(MAX_N);
+                }
+                return input;
+            });
+
+        state.files.inputWeatherFilePath.filePath = "in.epw";
+        app.add_option("-w,--weather", state.files.inputWeatherFilePath.filePath, "Weather file path (default: in.epw in current directory)")
+            ->required(false)
+            ->option_text("EPW")
+            ->check(CLI::ExistingFile);
```

Comment:
This uses a check to ensure the file exists. It'll fail right away if the passed argument isn't an existing file (or if not passed if the default "in.epw" doesn't exist in the current directory)